std/zip.d

```diff
@@ -283,7 +283,7 @@ final class ZipArchive
     import std.bitmanip : littleEndianToNative, nativeToLittleEndian;
     import std.algorithm : max;
     import std.conv : to;
-    import std.zlib : compress;
```

Comment:
The later use of `std.zlib.compress()` relies on a bug in dmd that messed up the scope list so that `std` was found. Import binds are supposed to prevent inserting the module package into the symbol table.